stable/database/templates/statefulset.yaml

```diff
@@ -173,6 +173,8 @@ spec:
         - { name: OVERWRITE_COPIES,    value: "{{ .Values.database.sm.logPersistence.overwriteBackoff.copies | default "3" }}" }
         - { name: OVERWRITE_WINDOW,    value: "{{ .Values.database.sm.logPersistence.overwriteBackoff.windowMinutes | default "120" }}" }
         - { name: SEPARATE_JOURNAL,    value: "{{- include "defaultfalse" .Values.database.sm.noHotCopy.journalPath.enabled}}" }
+        - { name: SNAPSHOT_RESTORED,   value: "{{ not (empty .Values.database.persistence.dataSourceRef) }}" }
+        - { name: BACKUP_ID,           value: "{{ .Values.database.autoImport.backup_id }}" }
```

Comment:
Why is it `backup_id` instead of `backupId`, which is more consistent with our current naming conventions?

Reply:
I chose the name to match the environment variable. You are right that it does not match the other helm values. Fixing.